wg-charter.html

```diff
@@ -0,0 +1,387 @@
+            </td>
+          </tr>
+          <tr id="Duration">
+            <th>
+              End date
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+
+          <tr>
+            <th>
+              Chairs
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Team Contacts
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Meeting Schedule
+            </th>
+            <td>
+              <strong>Teleconferences:</strong> as necessary.
+              <br />
+              <strong>Face-to-face:</strong> as necessary.
+            </td>
+          </tr>
+        </table>
+      </section>
+
+
+
+      <section id="scope" class="scope">
+        <h2>Scope</h2>
+        <p>
+            This Working Group will recommend a Web programming interface for
+            graphics and computation that:
+        </p>
+
+        <ol>
+            <li>enables rendering of modern graphics to both onscreen and offscreen drawing surfaces</li>
+            <li>enables computation tasks to be performed, and the results of such tasks to be retrieved</li>
+            <li>This group may also recommend a companion Shading Language that describes the graphics and computation tasks in a format
+            that can be translated or compiled into platform-specific instructions</li>
+        </ol>
+
+        <p>
+            The API will not be restricted to any particular platform technology. Instead, it will
+            be generic enough to be implemented on top of modern GPU libraries, such as
+            Microsoft's Direct3D 12, Apple's Metal, and Khronos's Vulkan.
+        </p>
+
+        <p>
+            Note that the majority of the input to this Working Group will come
+            directly from the <a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a>.
+            No major development will happen in the Working Group itself. Instead, the Community
+            Group will be driving the technical work.
+        </p>
+
+        <div>
+          <h3>Success Criteria</h3>
+          <p>In order to advance to  <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have at least two independent implementations of each of feature defined in the specification.</p>
+          <p>The WebGPU specification should contain a section detailing any <a href='https://www.w3.org/TR/security-privacy-questionnaire/'>known security</a>, fingerprinting, and privacy implications, and suggested <a href='https://w3c.github.io/perf-security-privacy/'>mitigation strategies</a> for implementers, Web authors, and end users. The group should not publish a specification if acceptable mitigation strategies cannot be found.</p>
```

Comment:
"The WebGPU specification **should** contain..." Well, will it? Might want to be a bit more certain here (**will** contain) and in the next paragraph.

Reply:
I'm more OK with the "**should not** publish a specification..." wording, but also wondering what you mean by "publish"; is this FPWD? WD updates, CR, or another PR requirement?

Reply:
I think this comment is still outstanding, but it seems to me that this text is more boilerplate for all W3C specifications than specific to WebGPU itself.

"should" gives options on how to organize such sections, while giving a clear goal to address these topics.

Reply:
Right, I stuck to the usual boilerplate text. The need to address (and seek review for) security, fingerprinting, privacy and other horizontals is a must in any case. I personally think it's ok to give some leeway for the group on ways to crystallizes the outcomes in the spec, but the text can be strengthened out if people prefer.